python/pyplugin_installer/installer.py

```diff
@@ -155,30 +155,30 @@ def checkingDone(self):
         plugins.rebuild()
         # look for news in the repositories
         plugins.markNews()
-        status = None
         # then check for updates (and eventually overwrite status)
-        updatable_count = 0
-        updatable_plugin_name = None
+        updatable_plugin_names = []
         for _, properties in plugins.all().items():
             if properties["status"] == "upgradeable":
-                updatable_count += 1
-                updatable_plugin_name = properties["name"]
+                updatable_plugin_names.append(properties["name"])
 
-        if updatable_count:
-            if updatable_count > 1:
-                status = self.tr("Multiple plugin updates are available")
-            else:
-                status = self.tr("An update to the {} plugin is available").format(updatable_plugin_name)
-            tabIndex = 3  # PLUGMAN_TAB_UPGRADEABLE
-
-        # finally set the notify label
-        if status:
-            bar = iface.messageBar()
-            self.message_bar_widget = bar.createMessage('', status)
-            update_button = QPushButton("Install Updates…")
-            update_button.pressed.connect(partial(self.showPluginManagerWhenReady, tabIndex))
-            self.message_bar_widget.layout().addWidget(update_button)
-            bar.pushWidget(self.message_bar_widget, Qgis.Info)
+        if len(updatable_plugin_names) <= 0:
```

Comment:
This can't possibly be < 0 :)
```suggestion
        if not updatable_plugin_names:
```